docs/pages/more/qr-codes.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: qr.expo.dev
+description: Reference for the QR code generator at qr.expo.dev.
+---
+
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+qr.expo.dev is a cloud function that generates Expo-branded QR codes. This function creates QR codes for EAS Updates, which are used to preview updates in development builds and Expo Go.
+
+For example, if you and your team have a development build, and you'd like to load the latest update on a specific build's channel, you could go to following endpoint to generate a QR code:
+
+```text
+https://qr.expo.dev/eas-update?projectId=your-project-id&runtimeVersion=your-runtime-version&channel=your-channel
+```
+
+Which would produce the following QR code SVG:
+
+<ImageSpotlight
+  alt="QR code generated by qr.expo.dev"
+  src="https://qr.expo.dev/eas-update?slug=your-slug&projectId=your-project-id&runtimeVersion=your-runtime-version&channel=your-channel"
+  style={{ maxWidth: 200 }}
+/>
+
+This QR code represents the following URL:
+
+```text
+exp+your-slug://expo-development-client/?url=https://u.expo.dev/your-project-id?runtime-version=your-runtime-version&channel-name=your-channel
+```
+
+This URL will deep link into a development build and instruct it to fetch the latest update on the specified channel.
+
+> **info** If sharing the URL is more convenient, you can request the URL directly by adding `format=url` to the query parameters.
+
+## General
+
+The following parameters apply to the /eas-update endpoint.
+
+### Base query parameters
+
+The following are base query parameters that can be included with any request to /eas-update.
+
+| param                    | required | default    | description                                                                                          |
+| ------------------------ | -------- | ---------- | ---------------------------------------------------------------------------------------------------- |
+| `slug`                   | no       | exp        | To target Expo Go, use "exp". Otherwise, use `slug` from **app.json** to target a development build. |
```

Comment:
```suggestion
| `slug`                   | No       | exp        | Use [`slug`](/versions/latest/config/app/#slug) from [app config](/workflow/configuration/) to target a development build. Otherwise, use "exp" to target Expo Go. |
```

Let's make the description development-build first and internal links. Also, instead of mentioning **app.json**, let's use app config as a term since it represents different variants of app config file regardless of the extension (such as app.config.js/app.config.ts).